source/docs/v3/integrations/livewire.blade.md

```diff
@@ -29,3 +31,49 @@ Now you can use Livewire both in the central app and the tenant app.
 Also make sure to enable [universal routes]({{ $page->link('features/universal-routes') }}).
 
 And if you're using file uploads, read the [Real-time facades]({{ $page->link('realtime-facades') }}) page of the documentation. Livewire uses real-time facades in the uploading logic.
+
+## Tenancy by path
+
+Open the `config/livewire.php` file and change this:
+
+```php
+'middleware_group' => ['web'],
+```
+
+to this:
+
+```php
+'middleware_group' => [
+    'web',
+    InitializeTenancyByPath::class,
+],
+```
+
+(Don't forget to import the middleware class.)
+
+Open the `config/livewire.php` file and add this:
```

Comment:
As @andreasmatu pointed out in https://github.com/archtechx/tenancy/issues/628#issuecomment-1294876192, this should be something like `routes/tenant.php`:
```suggestion
Open the `routes/tenant.php` file and add this:
```